vim/core/oni-core-interop/plugin/init.vim

```diff
@@ -113,3 +113,13 @@ function OniUpdateWindowDisplayMap()
 
     call OniNotify(["window_display_update", context, mapping])
 endfunction
+
+function OniApiInfo()
+    if (has_key(api_info(),'version'))
+        call OniNotify(["api_info",api_info()['version']])
+    else
+        call OniNotify(["api_info",{api_level:0}])
+    endif
+
```

Comment:
Could you please remove these extra newlines? No linting set up for us in VimL :)